unicodetools/src/main/java/org/unicode/text/UCD/GenerateData.java

```diff
@@ -1123,16 +358,13 @@ public static void writeNormalizerTestSuite(String directory, String fileName)
                     // firstCandidate and secondCandidate, look for strings that
                     // cannot be split between those two characters.
                     // Those are our test cases for Part 5.
-                    String nfc = Default.nfc().normalize(decomposition);
+                    String normalizedFormC = nfc.normalize(decomposition);
```

Comment:
optional: simply `normalized` or `normalizedC` ?

Reply:
Not `normalized`, since we have NFD flying around everywhere here. And I think I prefer the more verbose name.